crates/dht/src/dht.rs

```diff
@@ -98,8 +98,8 @@ pub trait DhtApi: 'static + Send + Sync + std::fmt::Debug {
         their_snapshot: DhtSnapshot,
         our_previous_snapshot: Option<DhtSnapshot>,
         arc_set: ArcSet,
-        max_op_data_bytes: u32,
-    ) -> BoxFut<'_, K2Result<(DhtSnapshotNextAction, u32)>>;
+        max_op_data_bytes: i32,
```

Comment:
The limit is now soft, so we might have a negative number in the limit if we've already exceeded it.

Reply:
I think we still want to return the next diff even if we're out of budget to return ops... Not certain about that but I think it still makes sense to proceed here.